bench/wtperf/wtperf.c

```diff
@@ -2349,21 +2349,23 @@ main(int argc, char *argv[])
 		/*
 		 * This is getting hard to parse.
 		 */
-		snprintf(cc_buf, req_len, "%s%s%s%s%s%s%s%s",
-		    cfg->conn_config,
+		snprintf(cc_buf, req_len, "%s%s%s%s%s%s%s",
 		    cfg->async_config ? cfg->async_config : "",
 		    cfg->compress_ext ? cfg->compress_ext : "",
-		    cfg->verbose > 1 ? ",": "",
-		    cfg->verbose > 1 ? debug_cconfig : "",
+		    cfg->verbose > 1 && strlen(debug_cconfig) ? ",": "",
+		    cfg->verbose > 1 &&
+			strlen(debug_cconfig) ? debug_cconfig : "",
 		    sess_cfg ? sess_cfg : "",
 		    user_cconfig ? ",": "",
 		    user_cconfig ? user_cconfig : "");
-		if ((ret = config_opt_str(cfg, "conn_config", cc_buf)) != 0)
-			goto err;
+		if (strlen(cc_buf))
+			if ((ret = config_opt_str(
+			    cfg, "conn_config", cc_buf)) != 0)
+				goto err;
```

Comment:
@sulabhM, @daveh86, just FYI: the WiredTiger parser discards extra separators, so a simpler approach to this is to use a config string like `%s,%s,%s,%s`, then you don't have to test whether or not additional commas are needed or have extra formats for the commas.